src/renderer/components/stake/AddStake/AddStake.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useCallback, useMemo, useState } from 'react'
+
+import { Asset, BaseAmount, baseAmount } from '@thorchain/asgardex-util'
+import { useIntl } from 'react-intl'
+
+import { PriceDataIndex } from '../../../services/midgard/types'
+import { AssetPair } from '../../../types/asgardex'
+import BigNumber from '../../uielements/assets/assetCard/AssetCard'
+import Drag from '../../uielements/drag'
+import * as Styled from './AddStake.style'
+
+type Props = {
+  asset: Asset
+  runeAsset: Asset
+  assetPrice: BigNumber
+  runePrice: BigNumber
+  assetAmount: BaseAmount
+  runeAmount: BaseAmount
+  priceIndex?: PriceDataIndex
+  unit?: string
+  assetData?: AssetPair[]
+  className?: string
+  onStake: (stakeAmount: BigNumber, asset: Asset) => void
+}
+
+export const AddStake: React.FC<Props> = ({
+  asset,
+  runeAsset,
+  assetPrice,
+  runePrice,
+  assetAmount,
+  runeAmount,
+  className,
+  assetData,
+  onStake
+  // priceIndex,
+  // unit,
```

Comment:
[nitpick] If we don't need `priceIndex` + `unit`, both can be removed from `Props`